Makefile

```diff
@@ -375,11 +375,6 @@ bundle-metrics: bundle-base ## Generate bundle manifests and metadata with metri
 	$(KUSTOMIZE) build config/manifests/metrics | $(OPERATOR_SDK) generate --verbose bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 	$(MAKE) bundle-validate
 
-.PHONY: bundle-metrics-ocp
```

Comment:
there still is bundle-build-metrics-ocp and container-build-metrics-ocp

Reply:
> removed also bundle-metrics

hm, IIRC the idea was that it can be used to manually create a k8s bundle some non OCP specific prometheus bits...  🤔

Reply:
other than that lgtm! 🙂

Reply:
oops, let me check again

Reply:
so, IIUC 

bundle-metrics + bundle-build-metrics + container-build-metrics

should to be restored (or at least the first two)